examples/compare_CPMG_echo.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+import pyspecdata as psd
+from pyspecProcScripts import lookup_table, select_pathway
+
+def echo_interleave(d, phcycdim):
+    "interleave even and odd echoes coming from phcycdim"
+    assert d.get_ft_prop(phcycdim)
+    retval = d.shape
+    retval.pop(phcycdim)
+    retval = retval.alloc(dtype=np.complex128, format=None)
+    retval["nEcho", 0::2] = d["ph1", +1]["nEcho", 0::2]
+    retval["nEcho", 1::2] = d["ph1", -1]["nEcho", 1::2]
+    retval.copy_axes(d).copy_props(d)
+    return retval
+
+with psd.figlist_var() as fl:
+    for thisfile, exp_type, nodename, manystep_cpmg, thislabel in [
+        (
+            "240702_13p5mM_TEMPOL_CPMG.h5",
+            "ODNP_NMR_comp/CPMG",
+            "CPMG_13",
+            False,
+            "CPMG simple cyc SW = 16",
+        ),
+        (
+            "240702_13p5mM_TEMPOL_CPMG.h5",
+            "ODNP_NMR_comp/CPMG",
+            "CPMG_19",
+            True,
+            "CPMG large cyc SW = 10.0",
+        ),
+        (
+            "240702_13p5mM_TEMPOL_echo.h5",
+            "ODNP_NMR_comp/Echoes",
+            "echo_12",
+            False,
+            "echo large cyc SW = 10.0",
+        ),
+    ]:
+        thisd = psd.find_file(
+            thisfile, exp_type=exp_type, expno=nodename, lookup=lookup_table
+        )
+        forplot = thisd.C.sum("nScans").ift("t2").run(abs)
+        thisd.squeeze()
+        fl.next("raw data for %s" % thislabel)
+        fl.image(thisd, interpolation="none")
```

Comment:
note that one of these plots doesn't show up properly.  you need to use the new `"auto"` interpolation